test/resources/language/erlang/rebar3/_checkouts/dependency/.gitignore

```diff
@@ -0,0 +1,17 @@
+.rebar3
+_*
+.eunit
+*.o
+*.beam
+*.plt
+*.swp
+*.swo
+.erlang.cookie
+ebin
+log
+erl_crash.dump
+.rebar
+logs
+_build
+.idea
+rebar.lock
```

Comment:
Both were generated by rebar3. I reduced them to things these integration tests actually create.